internal/api/service/health.go

```diff
@@ -4,41 +4,43 @@ import (
 	"time"
 
 	"github.com/base-org/pessimism/internal/api/models"
+	"github.com/base-org/pessimism/internal/client"
+	"github.com/base-org/pessimism/internal/core"
 	"github.com/base-org/pessimism/internal/logging"
+
 	"go.uber.org/zap"
 )
 
 // CheckHealth ... Returns health check for server
 func (svc *PessimismService) CheckHealth() *models.HealthCheck {
 	hc := &models.ChainConnectionStatus{
-		IsL1Healthy: svc.CheckETHRPCHealth(svc.cfg.L1RpcEndpoint),
-		IsL2Healthy: svc.CheckETHRPCHealth(svc.cfg.L2RpcEndpoint),
+		IsL1Healthy: svc.CheckETHRPCHealth(core.Layer1),
+		IsL2Healthy: svc.CheckETHRPCHealth(core.Layer2),
 	}
 
 	healthy := hc.IsL1Healthy && hc.IsL2Healthy
```

Comment:
A thought, is it worth parallelizing the health check here ? Would we be using this endpoint for aws health check ?

Reply:
+1; would definitely make the healthcheck respond faster

Reply:
#88